<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lot Details</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/medidoze-icon.ico</normaloff>../images/medidoze-icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background:#ffffff;
font-family:Nirmala UI
}
QGroupBox{
border:1px solid lightgrey;
font-weight: 700;
padding:10px 0px 0px 0px;
margin-top:8px;
font-family:Nirmala UI;
font-size:12pt;
border-radius:10%
}
QGroupBox:title{
subcontrol-origin: margin;
left: 10px;
top:-4px;
margin:0px 2px 0px 2px
}
QLabel{
font-size:10pt;
background:#ffffff;
}
QDateEdit{
padding:8px;
border:1px solid lightgrey;
border-radius:10%;
font-size:10pt
}
QDateEdit::drop-down{
image:url(:/icon/images/down_arrow.svg);
padding:17px 12px 17px 0px;
/*border:1px solid lightgrey;*/
}
QCalendarWidget QToolButton {
/* Style navigation buttons */
background-color: transparent;
color: black;
width:90px;
}
QCalendarWidget QToolButton:hover {
/* Hover effect for navigation buttons */
background-color: #e0e0e0;
}
 /* Hover on list of  month items */
QCalendarWidget QMenu::item:selected {
background-color:rgb(72, 201, 227);
color: white;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <property name="horizontalSpacing">
    <number>20</number>
   </property>
   <property name="verticalSpacing">
    <number>11</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>250</height>
      </size>
     </property>
     <property name="title">
      <string>Metadol</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-bottom:15px</string>
        </property>
        <property name="text">
         <string>Quantity (ml):</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="CustomDateEdit" name="expiryMetadol">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimumDate">
         <date>
          <year>2025</year>
          <month>8</month>
          <day>22</day>
         </date>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-bottom:15px</string>
        </property>
        <property name="text">
         <string>Lot No:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="txtQtyMetadol">
          <property name="styleSheet">
           <string notr="true">border-radius:10%;
padding:8px;
border:1px solid #e1e4e6;
font-size:10pt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="errQtyMetadol">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color:red;font-size:8pt</string>
          </property>
          <property name="text">
           <string>---</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="txtLotMetadol">
          <property name="styleSheet">
           <string notr="true">border-radius:10%;
padding:8px;
border:1px solid #e1e4e6;
font-size:10pt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="errLotMetadol">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color:red;font-size:8pt</string>
          </property>
          <property name="text">
           <string>---</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Expiry Date:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="infoMsgMetadol">
          <property name="styleSheet">
           <string notr="true">padding:8px</string>
          </property>
          <property name="text">
           <string>---</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="btnSaveLotMetadol">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ background:#48C9E3; color:#FFFFFF; padding:10px 25px; border-radius:6%;font-size:10pt; font-weight:700}</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>220</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777209</height>
      </size>
     </property>
     <property name="title">
      <string>Methadose</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Expiry Date:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="CustomDateEdit" name="expiryMethadose">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimumDate">
         <date>
          <year>2025</year>
          <month>8</month>
          <day>22</day>
         </date>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-bottom:15px</string>
        </property>
        <property name="text">
         <string>Quantity (ml):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-bottom:15px</string>
        </property>
        <property name="text">
         <string>Lot No:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="txtLotMethadose">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:10%;
padding:8px;
border:1px solid #e1e4e6;
font-size:10pt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="errLotMethadose">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color:red;font-size:8pt</string>
          </property>
          <property name="text">
           <string>---</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="txtQtyMethadose">
          <property name="styleSheet">
           <string notr="true">border-radius:10%;
padding:8px;
border:1px solid #e1e4e6;
font-size:10pt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="errQtyMethadose">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color:red;font-size:8pt</string>
          </property>
          <property name="text">
           <string>---</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="infoMsgMethadose">
          <property name="styleSheet">
           <string notr="true">padding:8px</string>
          </property>
          <property name="text">
           <string>---</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="btnSaveLotMethadose">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ background:#48C9E3; color:#FFFFFF; padding:10px 25px; border-radius:6%;font-size:10pt; font-weight:700}</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomDateEdit</class>
   <extends>QDateEdit</extends>
   <header>custom_date_edit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
